doc/gateway-instrumentation.md

```diff
@@ -0,0 +1,133 @@
+## Gateway instrumentation
+
+3scale Operator can enable insturmentation on it's managed APIcasts by using the [OpenTelemetry](https://opentelemetry.io/) SDK.
+More specifically, enabling the [Nginx opentelemetry tracing library](https://github.com/open-telemetry/opentelemetry-cpp-contrib/tree/main/instrumentation/nginx).
+
+It works with Jaeger since version **1.35**.  If the existing collector does not support
+OpenTelemetry traces, an OpenTelemetry Collector is required as tracing proxy.
+
+Supported propagation types: [W3C](https://w3c.github.io/trace-context/)
+
+### Prerequisites
+
+* Opentelemetry Collector supporting the APIcast exporter. Currently, the only implemeneted [exporter](https://opentelemetry.io/docs/reference/specification/protocol/exporter/)
+in APIcast is OTLP over gRPC `OTLP/gRPC`. Even though OpenTelemetry specification supports also OTLP over HTTP (`OTLP/HTTP`),
+this exporter is not included in APIcast. It works with Jaeger since version **1.35**.
+
+For dev/testing purposes, you can deploy quick&easy Jaeger with few commands. **Not suitable** for production use, though.
```

Comment:
Guess this answers my question above ,  We need Jaeger, but we don't need the deployment and make commands. I'm in two minds about this one , doen't install stuff we don't directly support  vs dev experience and make it easy :thinking:

Reply:
I think we can follow for now what was done in apicast operator. The jaeger command won't work unless the upstream repo is used (for dev/testing only) so I think we are safe.